src/bundle/Resources/public/js/scripts/core/tag.view.select.js

```diff
@@ -0,0 +1,144 @@
+(function(global, doc, eZ) {
+    class TagViewSelect {
+        constructor(config) {
+            this.inputSelector = config.inputSelector || 'input';
+            this.fieldContainer = config.fieldContainer;
+
+            if (!this.fieldContainer) {
+                throw new Error('Field Container doesn\'t exist!');
+            }
+
+            this.container = this.fieldContainer.querySelector('.ibexa-tag-view-select');
+            this.listContainer = this.container.querySelector('.ibexa-tag-view-select__selected_list');
+            this.inputField = this.fieldContainer.querySelector(this.inputSelector);
+            this.selectButton = this.container.querySelector('.ibexa-tag-view-select__btn-select-path')
+            this.singleSelect = this.container.dataset.singleSelect === '1';
+            this.canDeleteLast = this.container.dataset.canDeleteLast === '1';
```

Comment:
It was not so obvious to me what `canDeleteLast` does. Maybe `canBeEmpty` or something similar would be more precise? WDYT?